client/examples/ffi/src/test.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include "headers/casper_client.h"
+
+int main(int argc, char ** argv) {
```

Comment:
```suggestion
int main(int argc, char **argv) {
```


We should get a clang-format setup happening.

Reply:
It's a good suggestion, I'll look deeper into wiring that into a (not yet written) `Makefile`.